tests/concepts/test_manifest.py

```diff
@@ -0,0 +1,25 @@
+from uuid import UUID
+
+from poprox_storage.concepts.experiment import Experiment, Team
+from poprox_storage.concepts.manifest import manifest_to_experiment, parse_manifest_toml
+from poprox_storage.paths import project_root
+
+
+def test_load_manifest():
+    with open(project_root() / "tests" / "data" / "sample_manifest.toml") as f:
```

Comment:
` str / str` is new syntax to me. Does it just put in OS appropriate slashes?

Reply:
Basically, yes! `project_root()` returns a `pathlib.Path`, which provides this syntax and applies the OS-appropriate magic you inferred.